content/administration/install/cdn.rst

```diff
@@ -4,77 +4,98 @@ Setting up a Content Delivery Network (CDN)
+   ``pulltest-xxxxx.kxcdn.com``. This will differ for each database.
+
+Copy this :guilabel:`Zone URL` to a Notepad as it will be used in the next steps.
 
-In the Odoo back end, go to the :guilabel:`Website Settings`: menu, then
-activate the CDN support and copy/paste your zone URL in the
-:guilabel:`CDN Base URL` field. This field is only visible and configurable if
-you have developer mode activated.
+Step 2: Configure the Odoo instance with the new zone
+-----------------------------------------------------
 
-.. image:: cdn/odoo_cdn_base_url.png
-   :class: img-fluid
+In the Odoo Website App, go to the :menuselection:`Settings` and then activate the
+:guilabel:`Content Delivery Network (CDN)` and copy/paste the :guilabel:`zone URL` in the
+:guilabel:`CDN Base URL` field. This field is only visible and configurable when
+:doc:`../../applications/general/developer_mode` is activated.
 
-Now your website is using the CDN for the resources matching the
-:guilabel:`CDN filters` regular expressions.
+.. note::
+   Ensure that there are two "//"" *Forward Slashes* before the :guilabel:`CDN Base URL` and one "/"
+   *Forward Slash* after the :guilabel:`CDN Base URL`.
 
-You can have a look to the HTML of your website in order to check if the CDN
-integration is properly working.
+:guilabel:`Save` the Settings.
 
-.. image:: cdn/odoo_check_your_html.png
-   :class: img-fluid
+.. image:: cdn/cdn-base-url.png
+   :align: center
+   :alt: Activate the CDN setting in Odoo.
 
+Now the website is using the CDN for the resources matching the :guilabel:`CDN filters` regular
+expressions.
 
-Why should I activate CORS?
----------------------------
+In the HTML of the Odoo website the CDN integration is evidenced as working properly by checking the
+:abbr:`URL (Uniform Resource Locators)` of images. The CDN Base URL can be seen using the inspect
+function on the Odoo website.
 
-A security restriction in some browsers (Firefox and Chrome at time of writing)
-prevents a remotely linked CSS file to fetch relative resources on this same
-external server.
+.. image:: cdn/test-pull.png
+   :align: center
+   :alt: The CDN Base URL can be seen using the inspect function on the Odoo website.
 
-If you don't activate the CORS option in the CDN zone, the more obvious
-resulting problem on a default Odoo website will be the lack of font-awesome
-icons because the font file declared in the font-awesome CSS won't be loaded on
-the remote server.
+Why activate CORS?
+------------------
 
-Here's what you would see on your homepage in such a case:
+A security restriction in some browsers (Firefox and Chrome) prevents a remotely linked CSS file to
+fetch relative resources on this same external server.
 
-.. image:: cdn/odoo_font_file_not_loaded.png
-   :class: img-fluid
+If the CORS option isn't enabled in the :guilabel:`CDN zone`, the more obvious resulting problem on
+a default Odoo website will be the lack of font-awesome icons because the font file declared in the
```

Comment:
FA is a proper noun, no hyphen
```suggestion
a standard Odoo website will be the lack of Font Awesome icons because the font file declared in the
```

Reply:
Thanks for pointing this out!